packages/react/src/views/Message/Message.js

```diff
@@ -1,6 +1,6 @@
 import React, { memo, useContext } from 'react';
 import PropTypes from 'prop-types';
-import { format } from 'date-fns';
+import { format, set } from 'date-fns';
```

Comment:
importing set ?

Reply:
I think it was automatically imported by vs code . It doesn't have a use case in this context, I'll remove it.

Reply:
remove this too